code/game/objects/items/toys.dm

```diff
@@ -1442,3 +1443,205 @@ obj/item/toy/cards/deck/syndicate/black
 	else
 		playsound(loc, "swing_hit", 50, 1, -1)
 	..()
+
+/*
+ * Action Figures
+ */
+
+/obj/item/toy/figure
+	name = "Non-Specific Action Figure action figure"
+	desc = "A \"Space Life\" brand... wait, what the hell is this thing?"
+	icon = 'icons/obj/toy.dmi'
+	icon_state = "nuketoy"
+	var/cooldown = 0
+	var/toysay = "What the fuck did you do?"
+
+/obj/item/toy/figure/New()
+	desc = "A \"Space Life\" brand [name]"

```

Comment:
No idea if there is something actually important in the parents of this, but it probably should call & return parent anyways.